book/validation.rst

```diff
@@ -103,6 +103,11 @@ following:
     Protected and private properties can also be validated, as well as "getter"
     methods (see :ref:`validator-constraint-targets`).
 
+.. versionadded:: 2.7
+    Since 2.7, XML and YAML constraint files located in the `Resource/config/validation/`
+    sub-directory of a bundle are loaded. Previously, only `Resource/config/validation.xml`
+    and `Resource/config/validation.yml` were allowed to register contraints.
```

Comment:
We use a standardised sentence for the versionadded directives, can you please update this PR to follow it?

````rst
.. versionaded:: 2.7
    As of Symfony 2.7, XML and Yaml constraint files located in the
    ``Resources/config/validation`` sub-directory of a bundle are loaded. Prior
    to 2.7, only ``Resources/config/validation.yml`` (or ``.xml``) were loaded.
````